### ISSUE TITLE
Problem with Fanart

### ISSUE THREAD

**Issue by Kuddel071089** [instalator/ioBroker.kodi#51]
Using the Kodi Adapter in iobroker, i have problem with the Fanart.

I tested it with two films.

1st: "1917"
kodi.0.info.fanart = image://https%3a%2f%2fimage.tmdb.org%2ft%2fp%2foriginal%2fcqa3sa4c4jevgnEJwq3CMF8UfTG.jpg/
kodi.0.info.thumbnail = image://https%3a%2f%2fimage.tmdb.org%2ft%2fp%2foriginal%2f47oaeDJuu7U7e8eCFzNqfEj5piQ.jpg/

2nd; "Das Boot"
kodi.0.info.fanart = image://smb%3a%2f%2f192.168.4.10%2fData%2fFilme%2fSpielfilme%2fHD%2fDas%20Boot%20(1981)%2fDas.Boot.BDRip-fanart.jpg/
kodi.0.info.thumbnail = image://smb%3a%2f%2f192.168.4.10%2fData%2fFilme%2fSpielfilme%2fHD%2fDas%20Boot%20(1981)%2fDas.Boot.BDRip-poster.jpg/

The widgest of the Kodi Adapter only can handle with the picture paths of "1917" from the internet.
The pictures of "Das Boot" arent't shown.

I don't know why Kodi is using local pitcures at "Das Boot" and don't download them from the internet.

I've allready deleted my whole DB and all pictures on my NAS, with no succes. The problem is still there

Maybe you have an idea?

![1917](https://user-images.githubusercontent.com/34958991/97982906-bc0a5c80-1dd4-11eb-84aa-1b6521bd5530.png)

![das_boot](https://user-images.githubusercontent.com/34958991/97982907-bd3b8980-1dd4-11eb-988e-da18b42f851b.png)

**Comment by instalator** [comment]
The adapter does not download anything. It takes the link to the picture from the kodi itself.

**Comment by instalator** [comment]
Has the problem been resolved?

**Comment by Kuddel071089** [comment]
@instalator it is possible to fix the problem manually.
i had to set the picture for e.g. fanart diretly for the movie, becaus it wasn't set automatically.

Most of the time it works automatically for the other movies

**Comment by instalator** [comment]
Did you try to update your media library?

**Comment by Kuddel071089** [comment]
Yes i tried it multiple times, but the problem still exist. for the movie with the fanart failure, i fixed it now manually

**Comment by instalator** [comment]
kodi.0.info.thumbnail = image://music@nfs%3a%2f%2f192.168.88.11%2fvolume1%2fMusic%2fFlac%2f%d0%9c%d0%b0%d0%ba%d0%b0%d1%80%d0%b5%d0%b2%d0%b8%d1%87%20-%20%d0%9e%d1%82%20%d0%9c%d0%b5%d0%bd%d1%8f%20%d0%9a%20%d0%a2%d0%b5%d0%b1%d0%b5%2f01%20%d0%9d%d0%b0%d1%87%d0%b0%d0%bb%d0%be.m4a/

![image](https://user-images.githubusercontent.com/11523624/109937739-cba59d80-7d01-11eb-966e-ce577c562e88.png)
![image](https://user-images.githubusercontent.com/11523624/109938122-3f47aa80-7d02-11eb-8884-7d4e909de48a.png)

show how your links look like
